src/components/ContentHighlights/ContentHighlights.jsx

```diff
@@ -14,6 +14,15 @@ const ContentHighlights = () => {
   const [toasts, setToasts] = useState([]);
   const { enterpriseCuration: { enterpriseCuration } } = useContext(EnterpriseAppContext);
   useEffect(() => {
+    if (locationState?.highlightToast) {
```

Comment:
I kept the naming schema of `highlighToast` purposely since it does allow for more general purpose usage because of the string with no additional concatenations required. Previous implementations were specific to the toast's purpose, while this one can be reused anywhere in content highlights with customized messaging.